google/cloud/spanner_v1/proto/transaction.proto

```diff
@@ -28,9 +28,283 @@ option java_package = "com.google.spanner.v1";
 option php_namespace = "Google\\Cloud\\Spanner\\V1";
 option ruby_package = "Google::Cloud::Spanner::V1";
 
-// TransactionOptions are used to specify different types of transactions.
+// # Transactions
 //
-// For more info, see: https://cloud.google.com/spanner/docs/reference/rest/v1/Transaction
+//
+// Each session can have at most one active transaction at a time. After the
+// active transaction is completed, the session can immediately be
+// re-used for the next transaction. It is not necessary to create a
+// new session for each transaction.
+//
+// # Transaction Modes
```

Comment:
Headings are not prohibited in proto comments (specifically because it causes issues in client library doc generation) https://google.aip.dev/192#formatting

Reply:
I manually changed it to not contain headings during the migration so it wasn't blocked on changing the proto docstring. Until I am able to have the change made, I'll need to manually revert this docstring each time :-1: